src/pydvl/value/sampler.py

```diff
@@ -205,6 +200,24 @@ def weight(cls, n: int, subset_len: int) -> float:
         ...
 
 
+class StochasticSampler(PowersetSampler[T], abc.ABC):
```

Comment:
You just set `index_iteration`

Reply:
Remove inheritance from `PowersetSampler[T]`.